friendshipper/src/lib/components/playtests/PlaytestModal.svelte

```diff
@@ -290,26 +306,32 @@
 				value={playtest ? playtest.spec.feedbackURL : ''}
 			/>
 		</Label>
-		{#if mode === ModalState.Editing}
-			<Accordion>
-				<AccordionItem
-					class="p-2 hover:bg-secondary-700 dark:hover:bg-space-900"
-					activeClass="p-2 bg-secondary-700 dark:bg-space-900"
-					paddingDefault="p-2"
-				>
-					<span slot="header" class="text-base text-gray-300 flex gap-2">
-						<ExclamationCircleOutline class="mt-0.5" />
-						<span>Danger Zone</span>
-					</span>
-					<Button size="sm" color="red" disabled={deleting} on:click={() => handleDelete()}
-						>Delete Playtest
-					</Button>
-				</AccordionItem>
-			</Accordion>
-		{/if}
 		<Button type="submit" class="w-full" disabled={submitting}>Submit</Button>
 	</form>
 </Modal>
 
-<style>
-</style>
+<Modal
```

Comment:
one thing, since this confirmation modal, can be used pretty broadly
maybe this should become a ConfirmationModal under core/ui components